src/components/campaigns/DonorsAndDonations.tsx

```diff
@@ -56,28 +38,55 @@ const useStyles = makeStyles((theme: Theme) =>
   }),
 )
 
-export default function DonorsAndDonations() {
+export default function DonorsAndDonations({
+  donations,
+}: {
+  donations: CampaignDonation[] | undefined
+}) {
   const classes = useStyles()
   const { t } = useTranslation()
-
+  const [donationsToShow, setDonationsToShow] = useState<CampaignDonation[] | undefined>(undefined)
```

Comment:
You don't have to explicitly specify `undefined`, not passing anything will effectively be the same